projects/js-packages/charts/src/providers/chart-context/stories/index.docs.mdx

```diff
@@ -0,0 +1,553 @@
+}
+```
+
+## Accessibility
+
+### Theme Color Considerations
+
+When creating custom themes, ensure sufficient contrast between background and foreground colors to meet WCAG AA standards (4.5:1 for normal text, 3:1 for large text)
+
+### Automatic Color Generation
+
+When your data has more series than colors defined in your theme, the Global Charts Context automatically generates additional colors using a sophisticated algorithm:
+
+#### Golden Ratio Distribution
+
+The system uses the **golden ratio (φ ≈ 0.618)** to generate mathematically pleasing color distributions:
+
+- **Hue spacing** - New hues are distributed for optimal visual separation
+- **Hue range constraint** - Generated colors stay within an expanded range (130%) of your theme's existing colors
+- **Wrap-around handling** - Intelligently handles cases where theme colors span across the 0° hue boundary
+
+**Note that only hex colors are supported for generating colors.**
+
+#### Accessibility-First Generation
+
+All generated colors maintain WCAG AA compliance:
+
+- **Base lightness: 35%** - Ensures 4.5:1 contrast ratio against white backgrounds
+- **Lightness variations** - Creates 4 levels (35%, 43%, 51%, 59%) for variety while maintaining accessibility
+- **Saturation variations** - Uses 3 levels (60%, 75%, 90%) for visual distinction
+
+#### Perceptual Distance Optimization
+
+The algorithm ensures visual distinction between colors:
+
+- **Minimum distance threshold** - Maintains 25+ units of perceptual distance between colors
+- **Smart retry logic** - Up to 50 attempts per color to find sufficiently different options
+- **Distance calculation** - Uses weighted HSL distance optimized for chart readability
+
+#### Practical Example
+
+<Source
+	language="tsx"
+	code={ `// Theme with 3 colors
+	const limitedTheme = {
+		colors: [ '#006DAB', '#1F9828', '#FF8C8F' ]
+	};
+
+	// Data with 8 series - 5 additional colors will be auto-generated
+	const dataWithManySeries = [
+		{ label: 'Series 1', group: 'A' }, // Uses #006DAB
+		{ label: 'Series 2', group: 'B' }, // Uses #1F9828
+		{ label: 'Series 3', group: 'C' }, // Uses #FF8C8F
+		{ label: 'Series 4', group: 'D' }, // Auto-generated: ~hsl(193, 75%, 43%)
+		{ label: 'Series 5', group: 'E' }, // Auto-generated: ~hsl(76, 60%, 51%)
+		{ label: 'Series 6', group: 'F' }, // Auto-generated: ~hsl(318, 90%, 35%)
+		{ label: 'Series 7', group: 'G' }, // Auto-generated: ~hsl(201, 75%, 59%)
+		{ label: 'Series 8', group: 'H' }, // Auto-generated: ~hsl(84, 60%, 35%)
+	];
+
+	// All colors remain consistent across charts and re-renders
+	<GlobalChartsProvider theme={ limitedTheme }>
+		<LineChart data={ dataWithManySeries } />
+		<BarChart data={ dataWithManySeries } />
+	</GlobalChartsProvider>` }
+/>
+
+This ensures that even with limited theme colors, your charts maintain visual harmony, accessibility compliance, and consistent color assignments across your entire application.
+
+
+## Browser Compatibility
```

Comment:
This part doesn't feel necessary in the age of bundlers 🤔

Reply:
Done in [36d4986](https://github.com/Automattic/jetpack/pull/45313/commits/36d4986c68f184b61ca0d9792f5b6d3efcb4e8e7) and [6c7ac58](https://github.com/Automattic/jetpack/pull/45313/commits/6c7ac58f275caa339bbed1cfabf3302fb5ad3c98)